config/dev.exs

```diff
@@ -12,5 +12,27 @@ config :bgp, BGP.MyServer,
       asn: 64496,
       bgp_id: "172.16.1.4",
       host: "172.16.1.4"
+    ],
+    [
+      asn: 65_536,
+      bgp_id: "172.16.1.2",
+      host: "172.16.1.2",
+      transport: BGP.Server.Session.Transport.Process,
+      transport_opts: [server: BGP.MyOtherServer]
```

Comment:
What about:

```suggestion
      transport: [module: BGP.Server.Session.Transport.Process, opts: [server: BGP.MyOtherServer]]
```

We can group transport and its ops in the same configuration.

Reply:
Yes it's possible.